src/js/Events.js

```diff
@@ -491,16 +509,28 @@ function updateCindy() {
     csctx.restore();
 }
 
-function cs_keypressed(e) {
+function keyEvent(e, script) {
     var evtobj = window.event ? event : e;
     var unicode = evtobj.charCode ? evtobj.charCode : evtobj.keyCode;
     var actualkey = String.fromCharCode(unicode);
     cskey = actualkey;
     cskeycode = unicode;
-    evaluate(cscompiled.keydown);
+    evaluate(script);
     scheduleUpdate();
 }
 
+function cs_keydown(e) {
```

Comment:
Yes, they do to me as well. But they are consistent with existing code patterns, so if we are going to drop these, we should remove all the others as well.

They might help future improvements, though. Since all invocations of event scripts go through these functions, it's easy to add some code there. Adding debug output to one of these can be useful as well during development. So I don't have a strong opinion either way.